src/common/json_output.rs

```diff
@@ -143,3 +143,72 @@ impl EnclaveBuildInfo {
         EnclaveBuildInfo { measurements }
     }
 }
+
+/// The information about the signing certificate to be provided for a `describe-eif` request.
+#[derive(Clone, Serialize, Deserialize)]
+pub struct SignCertificateInfo {
+    #[serde(rename = "IssuerName")]
+    /// Certificate's subject name.
+    pub issuer_name: BTreeMap<String, String>,
+    #[serde(rename = "Algorithm")]
+    /// Certificate's signature algorithm
+    pub algorithm: String,
+    #[serde(rename = "NotBefore")]
+    /// Not before validity period
+    pub not_before: String,
+    #[serde(rename = "NotAfter")]
+    /// Not after validity period
+    pub not_after: String,
+}
+
+impl SignCertificateInfo {
+    /// Create new signing ceritificate information structure
+    pub fn new(
+        issuer_name: BTreeMap<String, String>,
+        algorithm: String,
+        not_before: String,
+        not_after: String,
+    ) -> Self {
+        SignCertificateInfo {
+            issuer_name,
+            algorithm,
+            not_before,
+            not_after,
+        }
+    }
+}
+
+/// The information to be provided for a `describe-eif` request.
+#[derive(Clone, Serialize, Deserialize)]
+pub struct DescribeEifInfo {
+    #[serde(rename = "EifVersion")]
+    /// EIF version.
+    pub version: u16,
+    #[serde(rename = "PCRs")]
```

Comment:
Can we make sure this field has the same name and format as the one outputed at build time.

Reply:
I think the easiest way would be to put a EnclaveBuildInfo here.